app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -1504,6 +1526,11 @@ interface PledgeFragmentViewModel {
             return shippingCost
         }
 
+        /**
+         * When retrieving the backing shipping information we do have available the LocationId on the backingObject
+         * and the list of shippingRules available for the selected reward, so we the backing shippingRule will be
```

Comment:
nit: change `so we the backing shippingRule will be` to `the backing shippingRule will be`